packages/web3-eth/test/integration/helper.ts

```diff
@@ -53,3 +55,72 @@ export const sendFewTxes = async ({
 	}
 	return res;
 };
+
+export const validateTransaction = (tx: TransactionInfo) => {
+	expect(tx.nonce).toBeDefined();
+	expect(tx.hash).toBeDefined();
+	expect(String(tx.hash)?.length).toBe(66);
+	expect(Number(tx.type)).toBe(0);
```

Comment:
This should be function argument instead. We can validate type2 transaction as well.